react/components/ProductPrice/index.js

```diff
@@ -155,21 +156,35 @@ const ProductPrice = (props, context) => {
               <IOMessage id={labelListPrice} />
             </div>
           )}
-          <Price
-            showPriceRange={showListPriceRange}
-            priceRange={listPriceRange}
-            price={listPrice}
-            rangeContainerClasses={classNames(
-              productPrice.listPriceValue,
-              listPriceRangeClass,
-              handles.price_listPriceRange,
-            )}
-            singleContainerClasses={classNames(
-              productPrice.listPriceValue,
-              listPriceClass,
-              handles.price_listPrice
-            )}
-          />
+          {!savingsAsPercentage &&
```

Comment:
Same thing here

Reply:
Because `savingsAsPercentage` toggles between the default way of showing savings *OR* show savings as percentage. It should not render both ways.